demos/http3/ossl-nghttp3-demo-server.c

```diff
@@ -130,9 +130,83 @@ static int on_end_stream(nghttp3_conn *h3conn, int64_t stream_id,
     return 0;
 }
 
-static int read_from_ssl_ids(nghttp3_conn *h3conn, struct h3ssl *h3ssl)
+/* Read from the stream and push to the h3conn */
+static int quic_server_read(nghttp3_conn *h3conn, SSL *stream, uint64_t id)
 {
+    int ret, r;
     uint8_t msg2[16000];
+    size_t l = sizeof(msg2) - 1;
```

Comment:
why minus 1 here?  we don't expect msg2 to be NULL terminated do we?

Reply:
fixed.